app/bundles/ProjectBundle/Service/ProjectEntityLoaderService.php

```diff
@@ -51,9 +52,11 @@ public function getProjectEntities(Project $project, array $entityTypes): array
                 ->getResult();
 
             $results[$entityType] = [
-                'label'    => $config->label,
-                'entities' => $entities,
-                'count'    => count($entities),
+                'label'       => $config->label,
+                'entities'    => $entities,
+                'count'       => count($entities),
+                'detailRoute' => $config->detailRoute
+        ?? new DetailRoute('mautic_'.$entityType.'_action', 'objectId', ['objectAction' => 'view']),
```

Comment:
```suggestion
                'detailRoute' => $config->detailRoute,
```
Let's not use a default detail route if it is not set. In theory in the future there might be an entity without detail page. Moreover this is a duplication of the [line number 211](https://github.com/mautic/mautic/pull/15493/files#diff-7581476e9d125cdd9761d1b4517a96fea4c558dcf994afca1b4ee4239c37fe56R211).